src/briefcase/platforms/android/gradle.py

```diff
@@ -241,20 +242,21 @@ def package_app(self, app: BaseConfig, **kwargs):
             "Building Android App Bundle and APK in release mode...",
             prefix=app.app_name,
         )
-        try:
-            self.subprocess.run(
-                # Windows needs the full path to `gradlew`; macOS & Linux can find it
-                # via `./gradlew`. For simplicity of implementation, we always provide
-                # the full path.
-                [self.gradlew_path(app), "bundleRelease"],
-                env=self.android_sdk.env,
-                # Set working directory so gradle can use the app bundle path as its
-                # project root, i.e., to avoid 'Task bundleRelease not found'.
-                cwd=self.bundle_path(app),
-                check=True,
-            )
-        except subprocess.CalledProcessError as e:
-            raise BriefcaseCommandError("Error while building project.") from e
+        with self.input.wait_bar("Building..."):
```

Comment:
Possibly "Bundling..." so it can't be confused with the build step?

Reply:
Updated.